scripts/publish/_publish_tools.sh

```diff
@@ -24,6 +24,11 @@ function publish_options {
   done
 }
 
+# Finds the version of the crate based on current working directory
+function crate_version_prefix {
+  grep -E '^version =' ./Cargo.toml | sed -e 's/.*"\([0-9]*\)\.\([0-9]*\)\.\([0-9]*\)"/\1_\2_\3/'
```

Comment:
`${REPO_ROOT}"/scripts/tools/cargo-dig.rs -v` will do this without the need for the regex. Makes it consistent with how we get the version in other spots of the automation.